lib/model/QubitInformationObject.php

```diff
@@ -567,6 +572,114 @@ public function canInheritRepository($repositoryId)
     return $repositoryId == $inheritedRepoId;
   }
 
+  /**
+   * Export or delete EAD and DC XML.
+   */
+  public function updateXmlExports()
+  {
+    // Create DC and EAD XML exports if the description's published... otherwise delete any that may exist
+    if ($this->getPublicationStatus()->statusId == QubitTerm::PUBLICATION_STATUS_PUBLISHED_ID)
+    {
+      $this->createXmlExports();
+    }
+    else
+    {
+      $this->deleteXmlExports();
+    }
+  }
+
+  /**
+   * Initate asynchronous jobs to export EAD and DC XML.
+   */
+  public function createXmlExports()
+  {
+    // Export top-level parent as EAD
+    $params = array(
+      'objectId' => $this->getCollectionRoot()->id,
+      'format'   => 'ead'
+    );
+
```

Comment:
Nitpick, is this blank line necessary? The block "// Export as DC" does not use it.